spec/models/category_organisation_spec.rb

```diff
@@ -4,4 +4,15 @@
   it 'should exist' do
     expect(CategoryOrganisation).not_to be_nil
   end
+
+  describe '#<=>' do
+    it 'use Category#<=> method to compare CategoryOrganisation' do
+      cat1 = FactoryGirl.create(:category, name: 'cat1')
+      cat2 = FactoryGirl.create(:category, name: 'cat2')
+      cat_org1 = CategoryOrganisation.create(category_id: cat1.id)
+      cat_org2 = CategoryOrganisation.create(category_id: cat2.id)
+      expect_any_instance_of(Category).to receive(:<=>).with(cat2).and_call_original
```

Comment:
For the unit test the `<=>` method is well covered via category model spec so to avoid duplicating the tests I added expectation that the method calls `<=>` on Category instances. 
I tried at first to use `expect(cat1).to receive(:<=>).with(cat2).and_call_original` but it didn't work 
Do you have some remarks for this test or it's acceptable ?